chore: update logo alt text (#1002) (#1014)
romefrontend/rome: README.md

<p align="center">

> 	<img alt="Rome, logo of an ancient roman arch colored yellow and white" src="https://github.com/romefrontend/rome/raw/main/assets/PNG/logo_transparent.png" width="700">

Reviewer: ```suggestion
	<img alt="Rome, logo of an ancient roman arch colored" src="https://github.com/romefrontend/rome/raw/main/assets/PNG/logo_transparent.png" width="700">
```

Author: I've seen alt texts on Twitter that go beyond that, but I don't have an authoritative answer. Maybe the color should only be added only when it's relevant to understanding the image, which is not the case either way.

```suggestion
	<img alt="Rome's logo depicitng a minimal ancient roman arch with the word Rome to its side" src="https://github.com/romefrontend/rome/raw/main/assets/PNG/logo_transparent.png" width="700">
```

Reviewer 2: Hey @smhmd, there is a typo at alt tag. The word depicitng should be depicting.

Reviewer 3: I think we should also be capitalizing the word "Roman" since it's a descriptive nationality in this context. Something like this, combining all suggestions so far:

```suggestion
	<img alt="Rome's logo depicting an ancient Roman arch with the word Rome to its side src="https://github.com/romefrontend/rome/raw/main/assets/PNG/logo_transparent.png" width="700">
```

Author: ```suggestion
	<img alt="Rome's logo depicting an ancient Roman arch with the word Rome to its side" src="https://github.com/romefrontend/rome/raw/main/assets/PNG/logo_transparent.png" width="700">
```
